src/Games/index.js

```diff
@@ -0,0 +1,23 @@
+import readlineSync from 'readline-sync';
+
+const numberOfRound = 3;
```

Comment:
Plz separate the logical parts of the code with empty lines. Now the code is hard to read

Reply:
ok, added, I hope again correct)